### ISSUE TITLE
BLorient577 - an error in the Répertoire... ?

### ISSUE THREAD

**Issue by MarcinKrawczuk** [BetaMasaheft/Documentation#1781]
In BLorient577 f. 53v (not online) there is a hymn to Liqānos (Wright p. 128, no. 14). In the database there is no hymn dedicated to this saint. But there is LIT3110RepCh390 which has the same incipit (or beginning of it, to be precise, because the incipit in Wright is longer than in the Répertoire).

So, my idea is that there is a mistake in the Répertoire and no. 390 is in fact dedicated to Liqānos, not Libānos. Actually if you look at the index of the Répertoire, no. 390 is listed under Liqānos. 

In short, I suggest that LIT3110RepCh390 is corrected so that it refers to Liqānos.
@thea-m @DariaElagina @antobrita

**Comment by DariaElagina** [comment]
Interesting! Alternatively, I would suggest, that you could encode the title with <certainty> if we can not by now check and be sure that LIT3110RepCh390 refers to Liqānos.

**Comment by thea-m** [comment]
I think that it would be good to edit the work entry to contain a reference to Liqānos, but also retain the one to Libanos, maybe with an explaining note.
